Source/Android/app/build.gradle

```diff
@@ -89,3 +92,7 @@ dependencies {
     // Allows FRP-style asynchronous operations in Android.
     compile 'io.reactivex:rxandroid:1.2.1'
 }
+
+def getVersion() {
+    return 'git describe --abbrev=0'.execute([], project.rootDir).text.trim()
```

Comment:
ok, updated to use the same command & regex used in the native code for version. this will show "5.0" for stable builds, "5.0-3312" for unstable